task4.py

```diff
@@ -0,0 +1,309 @@
+import os
+import csv
+import requests
+import logging
+import argparse
+from datetime import datetime, timedelta
+from collections import Counter
+import shutil
+import pytz
```

Comment:
don't abuse importing the whole modules. if you need only 1-2 features just import them